src/component/mxgraph/MxGraphRenderer.ts

```diff
@@ -65,16 +66,49 @@ export default class MxGraphRenderer {
       const bounds = shape.bounds;
       const parent = this.getParent(bpmnElement);
       const absoluteCoordinate = { x: bounds.x, y: bounds.y };
-      const style = this.computeStyleName(bpmnElement);
+      const style = this.computeStyleName(shape);
       this.insertVertexGivenAbsoluteCoordinates(parent, bpmnElement.id, bpmnElement.name, absoluteCoordinate, bounds.width, bounds.height, style);
     }
   }
 
-  private computeStyleName(bpmnElement: ShapeBpmnEvent | ShapeBpmnElement): string {
+  private computeStyleName(shape: Shape): string {
+    const bpmnElement = shape.bpmnElement;
+
     let style = bpmnElement.kind as string;
     if (bpmnElement instanceof ShapeBpmnEvent) {
       style += ';' + StyleConstant.BPMN_STYLE_EVENT_KIND + '=' + bpmnElement.eventKind;
     }
+    style += this.computeFontStyle(shape);
+
+    return style;
+  }
+
+  private computeFontStyle(bpmnCell: Shape | Edge): string {
+    let style = '';
+
+    const label = bpmnCell.label;
```

Comment:
you can use _optional chaining_ and instantiate font here directly to omit one of the 'if'
`const font = bpmnCell.label?.font;`
or if bpmnCell can be also null `const font = bpmnCell?.label?.font;`

Reply:
Done